src/main/java/com/stablekernel/rxespresso/RxIdlingResource.java

```diff
@@ -93,13 +98,12 @@ public void registerIdleTransitionCallback(ResourceCallback resourceCallback) {
     @Override
     public <T> Observable.OnSubscribe<T> onSubscribeStart(Observable<? extends T> observableInstance,
                                                           final Observable.OnSubscribe<T> onSubscribe) {
+
         int activeSubscriptionCount = subscriptions.incrementAndGet();
+        onSubscribeList.put(new WeakReference<Observable.OnSubscribe>(onSubscribe), new Throwable());
```

Comment:
weak ref so that we don't keep OnSubscribe objects from being garbage collected. Let whoever created this object release it.